pandas/tests/indexing/test_categorical.py

```diff
@@ -754,3 +762,76 @@ def test_map_with_dict_or_series(self):
         output = cur_index.map(mapper)
         # Order of categories in output can be different
         tm.assert_index_equal(expected, output)
+
+    @pytest.mark.parametrize(
+        "idx_values",
+        [
+            # python types
+            [1, 2, 3],
+            [-1, -2, -3],
+            [1.5, 2.5, 3.5],
+            [-1.5, -2.5, -3.5],
+            # numpy int/uint
+            *[
+                np.array([1, 2, 3], dtype=dtype)
+                for dtype in [
+                    np.int8,
+                    np.int16,
```

Comment:
any way to use the fixtures (or definitions) in pandas/conftest.py for some of this